test/integration/requests.test.ts

```diff
@@ -256,4 +256,98 @@ describe("/requests", function () {
         });
     });
   });
+
+  describe("GET /requests", function () {
+    it("should return all requests", function (done) {
+      chai
+        .request(app)
+        .get("/requests?dev=true")
+        .end(function (err, res) {
+          expect(res).to.have.status(200);
+          done();
+        });
+    });
+
+    it("should return all requests by specific user", function (done) {
+      chai
+        .request(app)
+        .get("/requests?dev=true&requestedBy=testUser")
+        .end(function (err, res) {
+          expect(res).to.have.status(200);
+          expect(res.body.data.every((e: any) => e.requestedBy === "testUser"));
+          done();
+        });
+    });
+
+    it("should return all requests by specific user and state", function (done) {
+      chai
+        .request(app)
+        .get("/requests?dev=true&requestedBy=testUser&state=APPROVED")
+        .end(function (err, res) {
+          expect(res).to.have.status(200);
+          expect(res.body.data.every((e: any) => e.requestedBy === "testUser" && e.state === "APPROVED"));
+          done();
+        });
+    });
+
+    // TODO: .skip to be removed
```

Comment:
Done my changes to return empty array with 200 status code , can you please remove these skips?

Reply:
status code that has to be send should be 204. Have added that comment also.

Reply:
done with 204 status code for no content but we can't send any message to the client